UnityProject/Assets/Scripts/Items/Cards/Emag.cs

```diff
@@ -118,7 +118,7 @@ private bool UseChargeLogic(GameObject Performer)
 				}
 				else
 				{
-					_ = SoundManager.PlayNetworkedForPlayer(recipient: Performer, OutOfChargesSFXA, sourceObj: gameObject);
+					SoundManager.PlayNetworkedForPlayer(recipient: Performer, OutOfChargesSFXA, sourceObj: gameObject);
```

Comment:
Doesn't this generate Codacy warnings?

Reply:
Nah, fire and forget for compiler warnings, but that method no longer is async.